Cargo.toml

```diff
@@ -13,9 +13,23 @@ edition = "2018"
 default-run = "electrs"
 
 [features]
-liquid = ["elements"]
-electrum-discovery = ["electrum-client"]
+liquid = [ "elements" ]
+electrum-discovery = [ "electrum-client"]
 bench = []
+default = ["no-otlp-tracing"]
+otlp-tracing = [
+    "tracing",
```

Comment:
The "tracing" crate can be made non-optional instead of enabling it in both cases.